### PR TITLE
Added description of chiliproject_migrate.rb

### PR DESCRIPTION
**Pull request by Codysseus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 # snippets
 This is a repo dedicated to snippets of code created to help with miscellaneous things.
+
+## chiliproject_migrate.rb
+This script is used for exporting everything related to a chiliproject project.
+To use this you first must from the project id of the project you are trying to migrate.
```
Comment (knightsamar) [inline README.md]: _obtain_ the project id.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
...
+The project id in the script must be replaced with the project id of the project you intend to export.
+This script creates the files in a directory specified by `$dump_dir` at the top of the script.
+Be sure to create this directory before running the script.
+Then, run `./script/console production` in the chiliproject root directory to gain access to the ORM console.
```
Comment (knightsamar) [inline README.md]: Actually, it's the standard Ruby REPL with all production settings loaded (and hence pointed to the production database)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+Once you have access, you paste the script in. Sure, it isn't pretty, but it works.
```
Comment (knightsamar) [inline README.md]: How do we determine 'access' ? :)

**Review by knightsamar — CHANGES_REQUESTED** [review]
Some small changes required and a bit of spacing, but good start!

File: README.md [hunk 4]
```diff
@@ -3,13 +3,17 @@ This is a repo dedicated to snippets of code created to help with miscellaneous
...
 Be sure to create this directory before running the script.
-Then, run `./script/console production` in the chiliproject root directory to gain access to the ORM console.
-Once you have access, you paste the script in. Sure, it isn't pretty, but it works.
-This will export all models related to the project into json fixtures.
-`$fileext` is the file extensions appended to the file names.
-`$encoder` is the function that is sent to the model to create the files.
-To get yml formatting, set this variable to `to_yaml`.
+Then, run `./script/console production` in the chiliproject directory to start the Ruby REPL with production settings loaded.
+If you can find your project, then you can proceed with the migration.
+If you can't find your project, make sure to check the settings in `chiliproject/config/database.yml`.
```
Comment (knightsamar) [inline README.md]: How do we determine whether we have found or not found our project on the REPL?

File: README.md [hunk 5]
```diff
@@ -3,13 +3,17 @@ This is a repo dedicated to snippets of code created to help with miscellaneous
...
+If these settings are correct, then make sure your version of Ruby is correct and that you are using the correct Gems.
+RVM is a great tool fror managing Ruby versions.
```
Comment (knightsamar) [inline README.md]: A link could be added to RVM installation page from this.

**Comment by Codysseus** [comment]
@knightsamar Changers made!

**Review by knightsamar — CHANGES_REQUESTED** [review]

**Comment by knightsamar** [comment]
Ah just realized, I had a review pending submission.

**Comment by Codysseus** [comment]
@knightsamar Made the changes!

**Comment by knightsamar** [comment]
@DesertBeagle all good, but can we have some paragraph breaks/line spacing added before we merge ? :)

**Comment by Codysseus** [comment]
@knightsamar What do you think of these changes?

**Review by knightsamar — APPROVED** [review]

**Pull request merged**